src/Microsoft.IdentityModel.Protocols/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,2 @@
+Microsoft.IdentityModel.Protocols.ConfigurationManager<T>.BackgroundTaskCancellationToken.get -> System.Threading.CancellationToken
```

Comment:
@jmprieur Heads up on this

Reply:
We need a way to shut down the background task. This seemed like the least invasive way. We could also add a shutdown method or make the type disposable.

Reply:
@jmprieur The most idiomatic way would be to make the type disposable. But it is little bit of a grey area as the type should be used as a singleton in almost all cases. We have to have some way to cancel the task. If you are okay with making this disposable, I am happy to make that change instead.

Reply:
Maybe a ShutdownBackgroundTask() method?

Reply:
or the cancellation token

Reply:
ShutdownBackgroundTask fits well here.